renpy.el

```diff
@@ -1981,7 +1987,84 @@ Use enclosing keywords to find image name symbol bounds.  Return a (BEG
 			 ;; TODO: Once we know the precise context, other
 			 ;; completion engines should not apply.
 			 :exclusive 'no)))))
-
+
+;;;; Flymake
+
+(defvar renpy--flymake-regex
+  (rx (or
+       ;; Example: path/to/script.rpy:111 Error message
+       (seq
+	bol
+	(group-n 1 (1+ (not (any ?: "\n"))))
+	":"
+	(group-n 2 (1+ digit))
+	" "
+	(group-n 3 (0+ (not "\n"))))
+       ;; Example: File \"path/to/file/script.rpy\", line 27: expected statement.
+       (seq
+	"File "
```

Comment:
Is it intentional that this second pattern does not match the beginning of the line, but the first pattern does?

Reply:
Good point, I checked error messages, and it makes sense to add bol here as well.